src/common/cpu.h

```diff
@@ -116,11 +117,11 @@ struct features {
 #elif ARCH_PPC
   bool altivec;
 #elif ARCH_ARM
-  bool NEON;  
+  bool NEON;
   bool SHA1;
   bool SHA2;
   bool CRC32;
-#endif  
```

Comment:
nit: Remove space

Reply:
about these 4 nit: remove space comments in `cpu.h`: My editor removed the existing trailing spaces. I can revert those if you prefer to address them separately,

Reply:
Ah, yes, that would be nice. I want to give this a format pass sometime soon.